src/encoded/commands/deploy.py

```diff
@@ -101,6 +15,26 @@
 from datetime import datetime
 
 import boto3
+from pathlib import Path
+
+
+REPO_DIR = f"{str(Path().parent.absolute())}"
+
+
+def _load_configuration(conf_path):
```

Comment:
Configuration for what?

Reply:
https://github.com/ENCODE-DCC/encoded/pull/3571#discussion_r503742678